make test seems to fail on master

1. **mount master inside the docker container**
```bash
docker run --rm -it -e GOFLAGS=-mod=vendor -v $(PWD):/go/src/github.com/terraform-providers/terraform-provider-aws -w /go/src/github.com/terraform-providers/terraform-provider-aws golang:1.13-stretch bash
```
2. **make tools**
```bash
root@67c172e86779:/go/src/github.com/terraform-providers/terraform-provider-aws# make tools
GO111MODULE=on go install ./awsproviderlint
GO111MODULE=on go install github.com/bflad/tfproviderdocs
GO111MODULE=on go install github.com/client9/misspell/cmd/misspell
GO111MODULE=on go install github.com/golangci/golangci-lint/cmd/golangci-lint
root@67c172e86779:/go/src/github.com/terraform-providers/terraform-provider-aws#
```

3. **make test**
```bash
root@67c172e86779:/go/src/github.com/terraform-providers/terraform-provider-aws# make test
==> Checking that code complies with gofmt requirements...
go test ./... -timeout=30s -parallel=4
?   	github.com/terraform-providers/terraform-provider-aws	[no test files]
ok  	github.com/terraform-providers/terraform-provider-aws/aws	(cached)
ok  	github.com/terraform-providers/terraform-provider-aws/aws/internal/flatmap	(cached)
ok  	github.com/terraform-providers/terraform-provider-aws/aws/internal/keyvaluetags	(cached)
ok  	github.com/terraform-providers/terraform-provider-aws/aws/internal/service/batch/equivalency	(cached)
ok  	github.com/terraform-providers/terraform-provider-aws/aws/internal/service/eks/token	(cached)
?   	github.com/terraform-providers/terraform-provider-aws/awsproviderlint	[no test files]
ok  	github.com/terraform-providers/terraform-provider-aws/awsproviderlint/passes	(cached)
panic: test timed out after 30s

goroutine 37 [running]:
testing.(*M).startAlarm.func1()
	/usr/local/go/src/testing/testing.go:1377 +0xdf
created by time.goFunc
	/usr/local/go/src/time/sleep.go:168 +0x44

goroutine 1 [chan receive]:
testing.(*T).Run(0xc000484400, 0x1084b96, 0xc, 0x10c3ce0, 0x48be66)
	/usr/local/go/src/testing/testing.go:961 +0x377
testing.runTests.func1(0xc000484300)
	/usr/local/go/src/testing/testing.go:1202 +0x78
testing.tRunner(0xc000484300, 0xc0002abdc0)
	/usr/local/go/src/testing/testing.go:909 +0xc9
testing.runTests(0xc0002ee4e0, 0x1a3f660, 0x2, 0x2, 0x0)
	/usr/local/go/src/testing/testing.go:1200 +0x2a7
testing.(*M).Run(0xc000480300, 0x0)
	/usr/local/go/src/testing/testing.go:1117 +0x176
main.main()
	_testmain.go:48 +0x135

goroutine 20 [select]:
go.opencensus.io/stats/view.(*worker).start(0xc000194870)
	/go/src/github.com/terraform-providers/terraform-provider-aws/vendor/go.opencensus.io/stats/view/worker.go:154 +0x100
created by go.opencensus.io/stats/view.init.0
	/go/src/github.com/terraform-providers/terraform-provider-aws/vendor/go.opencensus.io/stats/view/worker.go:32 +0x57

goroutine 21 [syscall]:
os/signal.signal_recv(0x0)
	/usr/local/go/src/runtime/sigqueue.go:147 +0x9c
os/signal.loop()
	/usr/local/go/src/os/signal/signal_unix.go:23 +0x22
created by os/signal.init.0
	/usr/local/go/src/os/signal/signal_unix.go:29 +0x41

goroutine 35 [syscall]:
syscall.Syscall6(0xf7, 0x1, 0x2b36, 0xc0000ec9f0, 0x1000004, 0x0, 0x0, 0x1, 0x2, 0xc0000eca30)
	/usr/local/go/src/syscall/asm_linux_amd64.s:44 +0x5
os.(*Process).blockUntilWaitable(0xc000576000, 0x40df56, 0x0, 0x2)
	/usr/local/go/src/os/wait_waitid.go:31 +0x98
os.(*Process).wait(0xc000576000, 0x10c4d30, 0x10c4d38, 0x10c4d28)
	/usr/local/go/src/os/exec_unix.go:22 +0x39
os.(*Process).Wait(...)
	/usr/local/go/src/os/exec.go:125
os/exec.(*Cmd).Wait(0xc0004ea000, 0x0, 0x0)
	/usr/local/go/src/os/exec/exec.go:501 +0x60
os/exec.(*Cmd).Run(0xc0004ea000, 0x310996d, 0x1a54980)
	/usr/local/go/src/os/exec/exec.go:341 +0x5c
golang.org/x/tools/go/packages.(*golistState).invokeGo(0xc0004e2000, 0x107d2fb, 0x4, 0xc0004e4000, 0x9, 0xe, 0x0, 0x0, 0x0)
	/go/src/github.com/terraform-providers/terraform-provider-aws/vendor/golang.org/x/tools/go/packages/golist.go:729 +0x4f0
golang.org/x/tools/go/packages.(*golistState).createDriverResponse(0xc0004e2000, 0xc000066030, 0x1, 0x1, 0xffffffffffffffff, 0xc000066020, 0xc00000e0a0)
	/go/src/github.com/terraform-providers/terraform-provider-aws/vendor/golang.org/x/tools/go/packages/golist.go:419 +0xb1
golang.org/x/tools/go/packages.goListDriver(0xc0004da008, 0xc000474570, 0x1, 0x1, 0x0, 0x0, 0x0)
	/go/src/github.com/terraform-providers/terraform-provider-aws/vendor/golang.org/x/tools/go/packages/golist.go:193 +0x441
golang.org/x/tools/go/packages.defaultDriver(0xc0004da008, 0xc000474570, 0x1, 0x1, 0xf0, 0xc00000c1e0, 0x44b5b7)
	/go/src/github.com/terraform-providers/terraform-provider-aws/vendor/golang.org/x/tools/go/packages/packages.go:229 +0x68
golang.org/x/tools/go/packages.Load(0xc0002fbdb8, 0xc000474570, 0x1, 0x1, 0x12, 0xc00000c1e0, 0xf, 0x1e, 0xdf6f20)
	/go/src/github.com/terraform-providers/terraform-provider-aws/vendor/golang.org/x/tools/go/packages/packages.go:214 +0x6f
golang.org/x/tools/go/analysis/analysistest.loadPackages(0xc0004a2230, 0x66, 0xc000474570, 0x1, 0x1, 0x1, 0x0, 0x1a55a40, 0x7f9dcd987d98, 0x0)
	/go/src/github.com/terraform-providers/terraform-provider-aws/vendor/golang.org/x/tools/go/analysis/analysistest/analysistest.go:144 +0x1e8
golang.org/x/tools/go/analysis/analysistest.Run(0x12ea800, 0xc000484400, 0xc0004a2230, 0x66, 0x1a464a0, 0xc000474570, 0x1, 0x1, 0xc00004cf60, 0x48be66, ...)
	/go/src/github.com/terraform-providers/terraform-provider-aws/vendor/golang.org/x/tools/go/analysis/analysistest/analysistest.go:106 +0xb3
github.com/terraform-providers/terraform-provider-aws/awsproviderlint/passes/AWSAT001_test.TestAWSAT001(0xc000484400)
	/go/src/github.com/terraform-providers/terraform-provider-aws/awsproviderlint/passes/AWSAT001/AWSAT001_test.go:12 +0xc4
testing.tRunner(0xc000484400, 0x10c3ce0)
	/usr/local/go/src/testing/testing.go:909 +0xc9
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:960 +0x350

goroutine 50 [IO wait]:
internal/poll.runtime_pollWait(0x7f9dcd8ebbf8, 0x72, 0xffffffffffffffff)
	/usr/local/go/src/runtime/netpoll.go:184 +0x55
internal/poll.(*pollDesc).wait(0xc00007a0d8, 0x72, 0x201, 0x200, 0xffffffffffffffff)
	/usr/local/go/src/internal/poll/fd_poll_runtime.go:87 +0x45
internal/poll.(*pollDesc).waitRead(...)
	/usr/local/go/src/internal/poll/fd_poll_runtime.go:92
internal/poll.(*FD).Read(0xc00007a0c0, 0xc000584000, 0x200, 0x200, 0x0, 0x0, 0x0)
	/usr/local/go/src/internal/poll/fd_unix.go:169 +0x1cf
os.(*File).read(...)
	/usr/local/go/src/os/file_unix.go:259
os.(*File).Read(0xc000010018, 0xc000584000, 0x200, 0x200, 0x0, 0x0, 0xc0005806a0)
	/usr/local/go/src/os/file.go:116 +0x71
bytes.(*Buffer).ReadFrom(0xc0004d8270, 0x12ea580, 0xc000010018, 0x7f9dcd8a7028, 0xc0004d8270, 0x1)
	/usr/local/go/src/bytes/buffer.go:204 +0xb4
io.copyBuffer(0x12e8b40, 0xc0004d8270, 0x12ea580, 0xc000010018, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/usr/local/go/src/io/io.go:388 +0x2ed
io.Copy(...)
	/usr/local/go/src/io/io.go:364
os/exec.(*Cmd).writerDescriptor.func1(0x0, 0x0)
	/usr/local/go/src/os/exec/exec.go:311 +0x63
os/exec.(*Cmd).Start.func1(0xc0004ea000, 0xc00000e100)
	/usr/local/go/src/os/exec/exec.go:435 +0x27
created by os/exec.(*Cmd).Start
	/usr/local/go/src/os/exec/exec.go:434 +0x608

goroutine 51 [IO wait]:
internal/poll.runtime_pollWait(0x7f9dcd8eba58, 0x72, 0xffffffffffffffff)
	/usr/local/go/src/runtime/netpoll.go:184 +0x55
internal/poll.(*pollDesc).wait(0xc00007a198, 0x72, 0x201, 0x200, 0xffffffffffffffff)
	/usr/local/go/src/internal/poll/fd_poll_runtime.go:87 +0x45
internal/poll.(*pollDesc).waitRead(...)
	/usr/local/go/src/internal/poll/fd_poll_runtime.go:92
internal/poll.(*FD).Read(0xc00007a180, 0xc000584200, 0x200, 0x200, 0x0, 0x0, 0x0)
	/usr/local/go/src/internal/poll/fd_unix.go:169 +0x1cf
os.(*File).read(...)
	/usr/local/go/src/os/file_unix.go:259
os.(*File).Read(0xc000010030, 0xc000584200, 0x200, 0x200, 0x0, 0x0, 0xc000580ea0)
	/usr/local/go/src/os/file.go:116 +0x71
bytes.(*Buffer).ReadFrom(0xc0004d82a0, 0x12ea580, 0xc000010030, 0x7f9dcd8a7028, 0xc0004d82a0, 0x1)
	/usr/local/go/src/bytes/buffer.go:204 +0xb4
io.copyBuffer(0x12e8b40, 0xc0004d82a0, 0x12ea580, 0xc000010030, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/usr/local/go/src/io/io.go:388 +0x2ed
io.Copy(...)
	/usr/local/go/src/io/io.go:364
os/exec.(*Cmd).writerDescriptor.func1(0x0, 0x0)
	/usr/local/go/src/os/exec/exec.go:311 +0x63
os/exec.(*Cmd).Start.func1(0xc0004ea000, 0xc00000e140)
	/usr/local/go/src/os/exec/exec.go:435 +0x27
created by os/exec.(*Cmd).Start
	/usr/local/go/src/os/exec/exec.go:434 +0x608

goroutine 52 [select]:
os/exec.(*Cmd).Start.func2(0xc0004ea000)
	/usr/local/go/src/os/exec/exec.go:443 +0xc4
created by os/exec.(*Cmd).Start
	/usr/local/go/src/os/exec/exec.go:442 +0x6a6
FAIL	github.com/terraform-providers/terraform-provider-aws/awsproviderlint/passes/AWSAT001	30.033s
FAIL
GNUmakefile:22: recipe for target 'test' failed
make: *** [test] Error 1
```

4. **make build**
```bash
root@67c172e86779:/go/src/github.com/terraform-providers/terraform-provider-aws# make build
==> Checking that code complies with gofmt requirements...
go install
```

--- a/GNUmakefile
+++ b/GNUmakefile
@@ -19,7 +19,7 @@ sweep:
 	go test $(TEST) -v -sweep=$(SWEEP) $(SWEEPARGS) -timeout 60m
 
 test: fmtcheck
-	go test $(TEST) -timeout=30s -parallel=4
+	go test $(TEST) -timeout=120s -parallel=4
 
 testacc: fmtcheck
 	TF_ACC=1 go test $(TEST) -v -count $(TEST_COUNT) -parallel 20 $(TESTARGS) -timeout 120m

